src/_guides/libraries/c-interop.md

```diff
@@ -184,27 +184,51 @@ while others (or their subtypes) can be instantiated.
+| [Float]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Float-class.html)                   | Represents a native 32 bit float in C.               |
+| [Int8]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Int8-class.html)                     | Represents a native signed 8 bit integer in C.       |
+| [Int16]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Int16-class.html)                   | Represents a native signed 16 bit integer in C.      |
+| [Int32]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Int32-class.html)                   | Represents a native signed 32 bit integer in C.      |
+| [Int64]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Int64-class.html)                   | Represents a native signed 64 bit integer in C.      |
+| [NativeFunction]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/NativeFunction-class.html) | Represents a function type in C.                     |
+| [Opaque]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Opaque-class.html)                 | The supertype of all opaque types in C.              |
+| [Uint8]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Uint8-class.html)                   | Represents a native unsigned 8 bit integer in C.     |
+| [Uint16]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Uint16-class.html)                 | Represents a native unsigned 16 bit integer in C.    |
+| [Uint32]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Uint32-class.html)                 | Represents a native unsigned 32 bit integer in C.    |
+| [Uint64]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Uint64-class.html)                 | Represents a native unsigned 64 bit integer in C.    |
+| [Void]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Void-class.html)                     | Represents a void type in C.                         |
+{:.table .table-striped }
+
+There are also many [ABI][] specific marker native types
+that extend [AbiSpecificInteger][].
+Refer to their linked API documentation for more information and
+a guideline on what types they map to on specific platforms:
+
+| **Dart type**                                                                                                 | **Description**                                              |
+|---------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|
+| [AbiSpecificInteger][]                                                                                        | The supertype of all ABI-specific integer types.             |
+| [Char]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Char-class.html)                         | Represents the C `char` type.                                |
```

Comment:
The wording is different in this table than the previous one.

"Represents a void type in C." versus "Represents the C `char` type."

Please make it consistent.